ikenna-portfolio/portfolio/src/test/java/com/google/sps/ServletFunctionsTest.java

```diff
@@ -42,12 +43,16 @@ public void tearDown() {
   public void testDataServletPost() throws Exception {
     HttpServletRequest request = mock(HttpServletRequest.class);       
     HttpServletResponse response = mock(HttpServletResponse.class);    
+    HttpSession session = mock(HttpSession.class);
 
     String name = "kenna";
     String text = "this some text";
+    User user = new User("example@google.com", false);
```

Comment:
Consider adding a test that the comment is not stored if there is no user info.

Reply:
done